metering/metering-installing-metering.adoc

```diff
@@ -5,5 +5,20 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Installing metering stub
+On OpenShift 4.x clusters the Metering Operator is available through Operator
+Hub and can be installed using the Operator Lifecycle Manager (OLM). If you are
+using an earlier version of OpenShift or a Kubernetes cluster without OpenShift,
+ use our upstream docs to install, configure, and use the Metering
```

Comment:
I had the same thought about including this note. I did see some instances of "upstream" in the repo. I think it would be helpful as metering is new and some people might want to hack on it, but I'm not against removing it if it causes clutter or goes against our principles.